### PR TITLE
CASMTRIAGE-2721: Update copyright and license headers

### PR DESCRIPTION
**Pull request by keopp-hpe**
## Summary and Scope

This PR updates the copyright and license headers in the uan-product-stream repo.

## Issues and Related PRs

* Resolves CASMTRIAGE-2721

### Test description:

Branch test build will need to made after the corresponding uan repo changes are built.

No content changes other than the copyright and license information

## Risks and Mitigations

Low risk

## Pull Request Checklist

- [X ] Version number(s) incremented, if applicable
- [X] Copyrights updated
- [ X] License file intact
- [ X] Target branch correct
- [ X] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -4,7 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
```
Comment (alanm-hpe) [inline ChangeLog.md:12]: It looks like a section was duplicated here.

File: docs/portal/developer-portal/css/H.css [hunk 2]
```diff
@@ -1,1268 +1,1293 @@
...
+*  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+*  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+*  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+*  THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+*  OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+*  ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+*  OTHER DEALINGS IN THE SOFTWARE.
+*
+ */
+/*
```
Comment (alanm-hpe) [inline docs/portal/developer-portal/css/H.css:26]: This diff looks a little weird. I also see two copyright headers now. Is there supposed to only be one?

Comment (alanm-hpe) [reply docs/portal/developer-portal/css/H.css:26]: I think I have the same comment for all of the *.css files. I'm not sure why the diff shows the entire file being removed and added and two copyrights are added at the top.

File: ChangeLog.md (hunk 1 above)
Comment (keopp-hpe) [reply ChangeLog.md:12]: Odd, but I removed the second copy now.  Push on the way shortly...

File: docs/portal/developer-portal/css/H.css (hunk 2 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/css/H.css:26]: I will fix this up.

File: ChangeLog.md (hunk 1 above)
Comment (keopp-hpe) [reply ChangeLog.md]: New commit with fixes pushed.

File: docs/portal/developer-portal/css/H.css (hunk 2 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/css/H.css]: Fixed *.css files and smoothed out some of the blank lines in the formatting of the file.

**Review by alanm-hpe — APPROVED** [review]

**Pull request merged**